docs/source/type_quals.rst

```diff
@@ -0,0 +1,340 @@
+Type qualifiers
+===============
+
+This section describes constructs that do not affect types of variables
+and methods, but affect how they can be accessed, assigned, and overridden.
+
+Class and instance variables
+****************************
+
+By default mypy assumes that a variable declared in the class body is
+an instance variable. One can mark names intended to be used as class variables
```

Comment:
> This is a bit confusing, and arguably not quite true. Mypy assumes that that it can used like either a class or an instance variable.

FWIW, this is the language used in PEP 526. When someone writes:
```python
class C:
    x: int

C.x = 0
```
variable `x` is an instance variable, and its _default_ can be set on class.